src/app/theme.ts

```diff
@@ -97,8 +97,10 @@ const color = {
 };
 
 const mobileSmallBreak = 30; // 480px
+const mobileLargerBreak = 50; // 800px
```

Comment:
maybe name it `mobileMadiumBreak` so we will have `small`, `medium` and "normal"